samples/bluetooth/peripheral_uart/Kconfig.sysbuild

```
@@ -6,8 +6,19 @@
 
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
+config NRF_IPC_RADIO_RPC_HOST
+	bool "Enable building RPC Host for Radio domain"
+	help
+	  The option enables building sample rpc_host application for Radio
+	  domain. This sysbuild configuration option should be selected only if
+	  an Application core application is build with CONFIG_BT_RPC_STACK
+	  enabled.
+
 config NRF_DEFAULT_IPC_RADIO
 	default y
```

```suggestion
	default y if !NRF_IPC_RADIO_RPC_HOST
```
This config enables the ipc_radio firmware on network core, so if you want to use rpc_host image as network core this option needs to be disabled.

---

After offline talk the sample was adjusted to use ipc_radio application for Radio domain + Bluetooth RPC_HOST subsystem.